TicketingContract.sol

```diff
@@ -224,12 +300,12 @@ contract TicketPro
         for(uint i = 0; i < ticketIndices.length; i++)
         {
             uint index = uint(ticketIndices[i]);
-            assert(inventory[_from][index] != bytes32(0));
+            assert(inventory[_from][index] != uint256(0));
             //pushes each element with ordering
             inventory[_to].push(inventory[msg.sender][index]);
             delete inventory[_from][index];
         }
-        
+
         emit TransferFrom(_from, _to, ticketIndices);
```

Comment:
Need to change the array in emitter to a count or the first ticketID. Arrays seem to produce garbage when you interpret the received return.